### PR TITLE
added schema tests and foundation tests

### PR DESCRIPTION
**Pull request by AlexSchwank**

### CODE REVIEW

File: test_assets/foundation/stage_configuration/metersPerUnit/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+![screenshot](screenshots/metersPerUnit_10_usdrecord_22.08.png)
+_metersPerUnit_10.usda, usdrecord 22.08_
+
+## MetersPerUnit mixed
+
+[metersPerUnit_mix.usda](./metersPerUnit_mixed.usda)
+
+This stage sets `metersPerUnit = 100` and also sublayers `metersPerUnit_1.usda` and `metersPerUnit_10.usda` with different values.
```
Comment (asluk) [inline test_assets/foundation/stage_configuration/metersPerUnit/README.md:31]: Is this intended to demonstrate a typical usage of mixed MPU, and/or to re-emphasize that USD itself won't handle any auto-scaling to reconcile divergent MPU across layer stacks? Would be great to call that out in the documentation; thanks!

**Review by meshula — COMMENTED** [review]
Per discussion in the wg meeting, we could add some documentation, or land it and add documentation subsequently. @AlexSchwank do you have an idea on how you'd like to proceed?

File: test_assets/foundation/stage_configuration/metersPerUnit/README.md (hunk 1 above)
Comment (AlexSchwank) [reply test_assets/foundation/stage_configuration/metersPerUnit/README.md:31]: I'll extend the documentation. The file is mainly aimed at showing that auto-scaling doesn't happen in the current implementation.

**Comment by AlexSchwank** [comment]
> Per discussion in the wg meeting, we could add some documentation, or land it and add documentation subsequently. @AlexSchwank do you have an idea on how you'd like to proceed?

I suggest we leave the PR open for another week and then merge it. This gives folks some time to point out things (like the MPU comment above) where more documentation would be good and I'll add it. Afterwards people can either create their own PRs or open issues to report missing/insufficient/out-of-date documentation.

**Comment by hybridherbst** [comment]
Some more questions of the more practical nature:
- are you planning to add tests for animations?
- are you planning (which would be great) to document somewhere which of these tests are working on which of the QuickLook implementations? (are all of them?)

**Comment by meshula** [comment]
The week has elapsed, Alex has updated with initial updates from comments. I think we are in a good place to start iterating, so I'm merging. @hybridherbst perhaps open an issue on animation tests, and another on documenting with tests are working in QuickLook?

**Comment by hybridherbst** [comment]
Created issues #16 and #17 - thanks!

**Pull request merged**